ui/app/AppLayouts/Profile/views/wallet/NetworksView.qml

```diff
@@ -86,6 +86,14 @@ Item {
             bottomPadding: Style.current.smallPadding
         }
 
+        Connections {
```

Comment:
@alaibe you don't need this at all, please revert all changed and try with this:
```qml
onClicked: testnetSwitch.onToggled()
components: [
StatusSwitch {
  id: testnetSwitch
  objectName: "testnetModeSwitch"
  checked: walletStore.areTestNetworksEnabled
  onToggled: {
    checked = Qt.binding(() => walletStore.areTestNetworksEnabled)
    Global.openTestnetPopup()
  }
}
]
```

Reply:
basically remove `checkable: false` and use `onToggled`

Reply:
I needed to keep the onClick otherwise clicking the row don't work